File: 005_number_to_ascii/number.rb
#!/usr/bin/env ruby

total_num = cur_num = 0
ARGF.each_with_index do |line, idx|
    num = line.to_i

    if idx == 0
        total_num = num
        next
	end

    cur_num += 1
    puts num.chr
end

File: 005_number_to_ascii/number.py
#!/usr/bin/python

import sys

total_num = cur_num = 0
for i, line in enumerate(sys.stdin):

  num = int(line)

  if i == 0:
    total_num = num
    continue
  else:
    cur_num += 1

  if cur_num > total_num:
    break 

  print chr(num)
